src/FcmChannel.php

```diff
@@ -52,21 +54,22 @@ public function send($notifiable, Notification $notification)
         if (count($tokens) == 1) {
             // Do not use multicast if there is only one recipient
             $fcmMessage->setTo(reset($tokens));
-            $this->sendToFcm($fcmMessage);
+            $responses[] = $this->sendToFcm($fcmMessage);
         } else {
             // Use multicast because there are multiple recipients
             $partialTokens = array_chunk($tokens, self::MAX_TOKEN_PER_REQUEST, false);
             foreach ($partialTokens as $tokens) {
                 $fcmMessage->setRegistrationIds($tokens);
-                $this->sendToFcm($fcmMessage);
+                $responses[] = $this->sendToFcm($fcmMessage);
             }
         }
+        return $responses;
```

Comment:
I don't think channels generally return a response.
If you want feedback, i think an event should be fired.

Reply:
Laravel fires NotificationSent and uses the return value from the send method as the response value - see https://github.com/laravel/framework/blob/6.x/src/Illuminate/Notifications/NotificationSender.php#L146-L150

Reply:
Example: https://github.com/laravel-notification-channels/apn/blob/a4f8bba8e111bf13617487923fae0eb02bfdb903/src/ApnChannel.php#L61

Reply:
Ahh, my bad. Seen it sent manually before, guess this is the right way!